example/ios/Podfile.lock

```diff
@@ -278,7 +278,7 @@ PODS:
   - Yoga (1.14.0)
 
 DEPENDENCIES:
-  - berbix-react-native (from `../.yalc/berbix-react-native`)
+  - berbix-react-native (from `../node_modules/berbix-react-native`)
```

Comment:
This seems like I might have done something wrong.

Reply:
hmmmm. I'm honestly not quite sure what to make of that. did you do an `npm i` at some point?

Reply:
Highly possible given recent investigations.

Reply:
A fresh clone and `yarn bootstrap` gives the same results.